index.html

```diff
@@ -2,24 +2,37 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="preconnect" href="https://fonts.googleapis.com/" />
-    <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300..900;1,300..900&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="preconnect" href="https://fonts.googleapis.com/" />
-    <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Redressed&display=swap"
-      rel="stylesheet"
-    />
-    <title>Vite + React + TS</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+<link
+  href="https://fonts.googleapis.com/css2?family=ABeeZee:ital@0;1&family=Redressed&family=Rubik:ital,wght@0,300;0,400;0,500;0,700;0,900;1,300;1,400;1,500;1,700;1,900&display=swap"
+  rel="stylesheet"
+/>
+    <title>Levantini</title>
+    <meta name="description" content="An engaging quiz app promoting peace and understanding through fun questions about diverse cultures. Celebrate diversity and unity by exploring the richness of global traditions, people, and communities." />
+    <meta name="keywords" content="quiz app, cultural diversity, religious diversity, famous people, trivia, Levantini, peace, understanding, global traditions, unity" />
+
+    
+    <meta name="theme-color" content="#ffff" /> 
+
+    <!-- Open Graph Tags -->
+    <meta property="og:title" content="Levantini" />
+    <meta property="og:description" content="An engaging quiz app promoting peace and understanding through fun questions about diverse cultures. Celebrate diversity and unity by exploring the richness of global traditions, people, and communities." />
+    <meta property="og:image" content="/src/logo.png" /> 
```

Comment:
Best practice for open graph images for social sharing is 1200x630 pixels. Lets perhaps take a small screen shot from figma of our website and put that as the image